EZP-32198 - update Perso docs/add screenshots (#97)
ezsystems/user-documentation: docs/personalization/scenarios.md

# Scenarios

A scenario is a configuration that is used to obtain recommendation results. 
It consists of:

- A content type to be returned as recommendation 
- A strategy (set of models) that is used for generating recommendations
- A filter configuration

If your [user Role](../site_organization/organizing_the_site.md#permissions) includes 
the `Personalization/View` Policy, you can view the list of scenarios and preview their result.

> To modify the scenarios to suit your needs, you must have the `Personalization/Edit` permissions.

Reviewer: ```suggestion
To modify the scenarios to suit your needs, you must have the `Personalization/Edit` Policy.
```

You can then navigate to the **Scenarios** tab and click the **Edit** icon next to 
a name of the scenario.

## Content type configuration

Every scenario supports a single input type and multiple output types. 
Every recommendation request delivers only content of one output content type 
(even if multiple types are selected in the interface below). 
The output type is set during the recommendation request and must be covered by 
the list of the supported content types in the requested scenario.

![Basic scenario configuration](img/scenario_configuration.png "Basic scenario configuration")

## Strategy configuration

> To modify the strategy, you drag model boxes between a list of all available models and the scenario 

Author: ```suggestion
To modify the strategy, you drag model boxes between a list of all available models and the strategy 
```

> matrix on the right side.

Author: ```suggestion
board on the right side.
```

To avoid empty or insufficient recommendation results, add several models to a strategy.

![Strategy configuration](img/scenario_configuration_strategy.png "Strategy configuration")

> You can arrange models within a strategy matrix by importance.

Author: ```suggestion
You can arrange models within a scenario board by importance.
```

> Models from each category are used in parallel and strategy results contain an equally 

Author: ```suggestion
Models from each level are used in parallel and strategy results contain an equally 
```
